waggle-dance-rest/pom.xml

```diff
@@ -98,6 +102,11 @@
       <artifactId>hamcrest-library</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.bundles.repackaged</groupId>
```

Comment:
What do we need this for? Also, this isn't marked with test scope but is in the are of the pom where all the test dependencies are.

Reply:
Its required for this: import jersey.repackaged.com.google.common.collect.Lists; used in FederationsAdminControllerTest. I have moved it to its correct location.

Reply:
Aah, that's an incorrect import, we should have spotted it earlier, it should just be `com.google.common.collect.Lists` (i.e. Guava directly instead of Jersey's relocated version). Please fix that import in the test and we can then remove this dependency.